package.json

```diff
@@ -43,14 +43,17 @@
     "prepare": "tsc"
   },
   "devDependencies": {
+    "@playwright/test": "^1.39.0",
```

Comment:
I think we should add another script that will run e2e test, because right now this command is only in github actions script: npx playwright test